ui-v2/app/components/tabular-collection.js

```diff
@@ -86,12 +120,15 @@ export default Component.extend(SlotsMixin, {
     window.removeEventListener('resize', this.handler);
   },
   resize: function(e) {
-    const $footer = [...$$('#wrapper > footer')][0];
-    const $thead = [...$$('main > div')][0];
-    if ($thead) {
-      // TODO: This should auto calculate properly from the CSS
-      this.set('height', Math.max(0, new Number(e.detail.height - ($footer.clientHeight + 218))));
-      this['cell-layout'] = new ZIndexedGrid($thead.clientWidth, 50);
+    const $tbody = this.$('tbody').get(0);
```

Comment:
I _think_ you could do `$$('tbody', this.element)[0]` here if you didn't want to mix 'n match jQuery & DOM stuff.

Reply:
👍 Ah nice yeah, will try that